webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -173,20 +183,24 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
 
   const fetchExperiments = useCallback(async (): Promise<void> => {
     try {
-      const tableOffset = Math.max((page - 0.5) * PAGE_SIZE, 0);
+      const tableOffset = Math.max((page - 1.5) * PAGE_SIZE, 0);
```

Comment:
nit: can you add some comments here and elsewhere you're doing calculations like this?